src/main/java/de/dennisguse/opentracks/viewmodels/IntervalStatisticsModel.java

```diff
@@ -103,6 +145,10 @@ public Distance getDistance(boolean metricUnits) {
                     .multipliedBy(multiplier);
         }
 
+        public boolean equals(IntervalOption intervalOption) {
+            return intervalOption != null && this.multiplier == intervalOption.multiplier;
```

Comment:
the method name is misleading as equals is otherwise used to check for equality with another object.